app/views/layouts/application.html.erb

```diff
@@ -5,41 +5,27 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <%#= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
```

Comment:
Kindly remove this unused code for it makes your code base look untidy and may cause you to have errors.  Please do this in all files were you have code commented out.